ModelFactoryCollection/ModelFactoryCollection.php

```diff
@@ -71,10 +71,17 @@ public function createModels($objects)
         // If that fails, try to find model factory for each item individually.
         $models = [];
         foreach ($objects as $key => $object) {
+            if (!is_object($object)) {
```

Comment:
I was thinking about this change a bit more and maybe it is a bit limiting to require object here. Let's say one would like to send array of JSON strings to model factory, that would internally convert them to some objects. This would seem to be a potentially interesting use case that would be blocked by this change.

Maybe instead I could move this condition to line 83 (https://github.com/xsolve-pl/xsolve-model-factory-bundle/pull/7/files#diff-2cb61b3427ed7229290e30d5058e70eeR83) so that exception message is clear?